crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -98,3 +104,17 @@ const transform = (
 };
 
 export { transform as default };
+
+function makeErrorEmitter(severity: "warning" | "error", ipc: Ipc) {
+  return function (error: Error | string) {
+    ipc.send({
+      type: "emittedError",
+      severity: severity,
+      error: {
+        name: error instanceof Error ? error.name : "Error",
+        message: error instanceof Error ? error.message : error,
```

Comment:
Maybe this is the bug here?
```suggestion
        message: error instanceof Error ? error.message : String(error),
```